examples/time_series/goes_xrs_example.py

```diff
@@ -97,7 +97,7 @@
 goes_flare = goes_15.truncate("2015-06-21 09:35", "2015-06-21 10:30")
 
 time_support()
-fig, ax = plt.subplots()
+fig, ax = plt.subplots(figsize=(10, 8))
```

Comment:
The bottom time axis text was cut off, not sure if this renders nicely on rTD, I need to check